src/pages/advance/clerk-idp.mdx

```diff
@@ -0,0 +1,72 @@
+import { Callout } from "nextra-theme-docs";
+import { Images } from "@/components/Images";
+
+# Use Clerk as an Identity Provider 
+
+Learn how to use Clerk to facilitate Single Sign-On (SSO) with other clients that support the OAuth 2.0 protocol. 
+
+&nbsp;  
+
+
+<Images
+  width={2279}
+  height={1135}
+  src="/images/advance/clerk-idp-flow-2.svg"
+  alt="Clerk IDP Flow"
+/>
+
+
+## Step 1: Retrieve Callback URL from Client
+
+From the client that you want to configure, retrieve the **Callback URL**. This is where your users will be redirected to after a successful authentication and is required to successfully create your OAuth application in the next step.
```

Comment:
```suggestion
Retrieve the **Callback URL** for the client application you want to configure. This is where your users will be redirected to after a successful authentication and is required to successfully create your OAuth application in the next step.
```

Reply:
I'd also recommend doing a quick pass across popular SaaS that we use and offer OAuth2 SSO as well as competitor OAuth2 SSO products (Auth0, Ory Hydra, etc...) to check the naming and inform our users that the name might differ. 
Variations may include terms such as callback url, redirect url, authorize url, etc...